### PR TITLE
OCSA hack club website

### PR DESCRIPTION
**Pull request by BrightTheBackpack**
This is the website for OCSA's Hack Club.  We hope to make this a hub for our clubs resources, including meeting slides and contact information.

### CODE REVIEW

**Review by grymmy — CHANGES_REQUESTED** [review]
@BrightTheBackpack Please provide a good high-level description to this PR stating what its intended goal is.

**Comment by BrightTheBackpack** [comment]
> @BrightTheBackpack Please provide a good high-level description to this PR stating what its intended goal is.

wrote a desscription, hope its good

File: hackclub.com.yaml [hunk 1]
```diff
@@ -1642,6 +1642,12 @@ nust:
   - ttl: 1
     type: CNAME
     value: nusthackclub.netlify.app.
+ocsa:
+  - ttl: 1
+    type: TXT
+    values:
+      - google-site-verification=O2LlohyBOM3x0c_m9P7sn14o12H802WsaNle7XIY3QE
+
```
Comment (Muirrum) [inline hackclub.com.yaml:1650]: Looks like you're missing a CNAME record -- this won't direct anyone to your site, just verify that you have the subdomain.

**Review by grymmy — APPROVED** [review]
Please work with @Muirrum to land this PR from hereforth.

File: hackclub.com.yaml (hunk 1 above)
Comment (Muirrum) [reply hackclub.com.yaml:1650]: Yup, you can still use TXT records, but this looks incomplete. A TXT record won't actually connect `ocsa.hackclub.com` to Google Sites.

```suggestion
ocsa:
  - ttl: 1
    type: TXT
    values:
      - google-site-verification=O2LlohyBOM3x0c_m9P7sn14o12H802WsaNle7XIY3QE
  - ttl: 300
    type: CNAME
    values:
      - ghs.googlehosted.com.
```

This suggestion should work -- see https://www.ionos.ca/digitalguide/domains/domain-tips/google-sites-with-custom-domain/ for a source

Comment (BrightTheBackpack) [reply hackclub.com.yaml:1650]: sure we can try that

Comment (BrightTheBackpack) [reply hackclub.com.yaml]: Seems like it didn't pass the test

Comment (BrightTheBackpack) [reply hackclub.com.yaml]: I think we should try with just the txt record, and if it doesn't work, I can make further pull requests. I think how it works is google sites first verifies the Domain with the txt record, them provides instructions for a cname or something else

File: hackclub.com.yaml [hunk 2]
```diff
@@ -1642,6 +1642,15 @@ nust:
...
     value: nusthackclub.netlify.app.
+ocsa:
+  - ttl: 1
+    type: TXT
+    values:
+      - google-site-verification=O2LlohyBOM3x0c_m9P7sn14o12H802WsaNle7XIY3QE
+  - ttl: 300
+    type: CNAME
+    values:
+      - ghs.googlehosted.com.
```
Comment (Muirrum) [inline hackclub.com.yaml:1653]:
```suggestion
ocsa:
  - ttl: 1
    type: TXT
    value:
      - google-site-verification=O2LlohyBOM3x0c_m9P7sn14o12H802WsaNle7XIY3QE
  - ttl: 300
    type: CNAME
    value:
      - ghs.googlehosted.com.
```

Whoops! That's my bad! octodns expects `value`, not `values`.

**Review by Muirrum — CHANGES_REQUESTED** [review]

**Review by Muirrum — APPROVED** [review]

**Comment by BrightTheBackpack** [comment]
will this be merged soon?

**Pull request merged**